copilot-chat-body.el

```diff
@@ -32,6 +32,50 @@
 (require 'copilot-chat-instance)
 (require 'copilot-chat-prompts)
 
+(defcustom copilot-chat-use-instruction-files t
+  "Use custom instructions from `.github/copilot-instructions.md'."
+  :type 'boolean
+  :group 'copilot-chat)
+
+(defcustom copilot-chat-max-instruction-size 65536
+  "Maximum size in bytes of `.github/copilot-instructions.md'."
+  :type '(choice
+          (const :tag "Unlimited" nil)
+          integer)
+  :group 'copilot-chat)
+
+(defun copilot-chat--read-instructions-file ()
+  "Return the content of `.github/copilot-instructions.md' or nil.
+If the file is larger than `copilot-chat-max-instruction-size',
+ignore it and emit a message."
+  (let* ((dir (or (and buffer-file-name (file-name-directory buffer-file-name))
+                  default-directory))
+         (root (locate-dominating-file dir ".github"))
```

Comment:
`locate-dominating-file` can use a file as start point so you can use `buffer-file-name` or `default-directory` directly.

Reply:
I fixed.